### DISCUSSION TITLE
Cannot access the security onion web interface on my windows 19 server

### DISCUSSION THREAD

**Discussion by JeffreyMunro** [Security-Onion-Solutions/securityonion#7496] [General]
I downloaded sysmon fine, and now am trying to download winlogbeat. When I try to go to security onions interface on my windows server VM I don't have access. I thought maybe I just need to give it access with so allow, and give the machine also "a" (analyst) so now it has "a" and "b" on the server IP. 

Although, when I try to connect to the page I still can't connect to it. Am I missing something? I am unsure, and still pretty new to this and would love feedback or help finding my solution.

I did another attempt this is what happened:
I checked everything with "sudo so-allow-view" and the IP is listed.  

When I tried to connect this time, it went to the page slowly, and briefly hardly responded. I switched browsers just to see, and am using firefox now, I accepted the risk and then the connection timed out and is continuing to timeout.

I also have an ubuntu machine, which ip has access and can can get to the web interface fast and easy. I'm new to homelabbing, but i'm trying to learn. I would love some feedback and tips. If there are any questions, I can go into it more and hopefully find a solution!

**Comment by dougburks** [comment]
Are the machines in the same subnet?

Are there any network firewalls between the machines?

Can you ping between the machines?

When you configured Security Onion and it asked how you would like to access it, did you choose IP, hostname, or other?
